Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -706,6 +706,40 @@ lemma integrable_add_iff_integrable_left {f g : α → β} (hf : Integrable f μ
     Integrable (g + f) μ ↔ Integrable g μ := by
   rw [add_comm, integrable_add_iff_integrable_right hf]
 
+lemma integrable_left_of_integrable_add_of_nonneg {f g : α → ℝ}
+    (h_meas : AEStronglyMeasurable f μ) (hf : 0 ≤ᵐ[μ] f) (hg : 0 ≤ᵐ[μ] g)
+    (h_int : Integrable (f + g) μ) : Integrable f μ := by
+  simp_rw [Integrable, h_meas, true_and]
```

Comment:
Can you prove this more directly using [MeasureTheory.Integrable.mono](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Function/L1Space.html#MeasureTheory.Integrable.mono)?

Reply:
Yes, the proof becomes much shorter, thank you!